mongodb/src/test/scala/docs/scaladsl/MongoSourceSpec.scala

```diff
@@ -36,9 +37,6 @@ class MongoSourceSpec
   implicit val system = ActorSystem()
   // #init-system
 
-  override protected def beforeAll(): Unit =
```

Comment:
This is not required anymore since every test run creates a fresh docker instance with empty state.